traffic_router/connector/src/main/java/com/comcast/cdn/traffic_control/traffic_router/secure/CertificateRegistry.java

```diff
@@ -17,14 +17,29 @@
 
 import com.comcast.cdn.traffic_control.traffic_router.protocol.RouterNioEndpoint;
 import com.comcast.cdn.traffic_control.traffic_router.shared.CertificateData;
+//import com.fasterxml.jackson.core.type.TypeReference;
```

Comment:
these imports can be removed if they're usused

Reply:
done